File: samples/java_springboot/54.teams-task-module/src/main/java/com/microsoft/bot/sample/teamstaskmodule/TaskModuleUIConstants.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.bot.sample.teamstaskmodule;

public final class TaskModuleUIConstants {
    private TaskModuleUIConstants() {

    }

    public static final UISettings YOUTUBE = new UISettings(
        1000,
        700,
        "YouTube Video",
        TaskModuleIds.YOUTUBE,
        "YouTube"
    );

    public static final UISettings CUSTOMFORM = new UISettings(
        510,
        450,
        "Custom Form",
        TaskModuleIds.CUSTOMFORM,
        "Custom Form"
    );

    public static final UISettings ADAPTIVECARD = new UISettings(
        400,
        200,
        "Adaptive Card: Inputs",
        TaskModuleIds.ADAPTIVECARD,
        "Adaptive Card"
    );
}

File: archive/samples/python/54.teams-task-module/models/task_module_ui_constants.py
# Copyright (c) Microsoft Corp. All rights reserved.
# Licensed under the MIT License.

from .ui_settings import UISettings
from .task_module_ids import TaskModuleIds


class TaskModuleUIConstants:
    YOUTUBE = UISettings(1000, 700, "YouTube Video", TaskModuleIds.YOUTUBE, "YouTube")
    CUSTOM_FORM = UISettings(
        510, 450, "Custom Form", TaskModuleIds.CUSTOM_FORM, "Custom Form"
    )
    ADAPTIVE_CARD = UISettings(
        400, 200, "Adaptive Card: Inputs", TaskModuleIds.ADAPTIVE_CARD, "Adaptive Card",
    )
